pkgs/development/libraries/ndn-cxx/default.nix

```diff
@@ -1,24 +1,36 @@
-{ stdenv, fetchFromGitHub, openssl, doxygen
-, boost, sqlite, pkgconfig, python, pythonPackages, wafHook }:
-let
-  version = "0.6.3";
-in
-stdenv.mkDerivation {
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, wafHook
+, openssl
+, doxygen
+, boost
+, sqlite
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
   pname = "ndn-cxx";
-  inherit version;
+  version = "0.7.1";
+
   src = fetchFromGitHub {
     owner = "named-data";
     repo = "ndn-cxx";
-    rev = "a3bf4319ed483a4a6fe2c96b79ec4491d7217f00";
-    sha256 = "076jhrjigisqz5n8dgxwd5fhimg69zhm834m7w9yvf9afgzrr50h";
+    rev = "${pname}-${version}";
```

Comment:
Isn't that $name or was that combined with ``_``?

Reply:
I want to keep this explicit for GitHub revision